task/control-plane-agent/src/main.rs

```diff
@@ -585,25 +589,10 @@ impl NetHandler {
 }
 
 fn sp_port_from_udp_metadata(meta: &UdpMetadata) -> SpPort {
-    use task_net_api::VLAN_RANGE;
-    assert!(VLAN_RANGE.contains(&meta.vid));
-    assert_eq!(VLAN_RANGE.len(), 2);
-
-    match meta.vid - VLAN_RANGE.start {
-        0 => SpPort::One,
-        1 => SpPort::Two,
-        _ => unreachable!(),
-    }
-}
-
-#[allow(dead_code)]
-fn vlan_id_from_sp_port(port: SpPort) -> u16 {
-    use task_net_api::VLAN_RANGE;
-    assert_eq!(VLAN_RANGE.len(), 2);
-
-    match port {
-        SpPort::One => VLAN_RANGE.start,
-        SpPort::Two => VLAN_RANGE.start + 1,
+    use task_net_api::VLANS;
+    match VLANS.iter().find(|v| v.vid == meta.vid).unwrap().port {
```

Comment:
Please use `unwrap_lite` if this must panic.

Reply:
Obviated in the Grand VLAN Refactoring